config/initializers/statsd.rb

```diff
@@ -235,6 +235,11 @@
 
 StatsD.increment('iam_ssoe_oauth.auth_type', 0)
 
+%w[IDME MHV DSL].each do |cred|
```

Comment:
Can you reference the STATSD_OAUTH_SESSION_KEY constant here?

Reply:
Done